AirCasting/SessionViews/SessionCartViewModel/DefaultDeleteSessionViewModel.swift

```diff
@@ -74,6 +77,16 @@ class DefaultDeleteSessionViewModel: DeleteSessionViewModel {
             } catch {
                 Log.info("Error when deleting sessions/streams")
             }
+            if sessionSynchronizer.syncInProgress.value {
```

Comment:
I think this could be extracted to a separate function. Additionally, I think if could be changed to guard, which indicates early return.

Reply:
Yeah, great catch - thanks! I've changed both then 😁